routes/users_routes.js

```diff
@@ -1,11 +1,10 @@
 const app = require('express').Router()
 const usersController = require('../controllers/users_controller')
+usersController.setUp()
```

Comment:
shouldn't something like this go inside `index.js`

Reply:
essentially the same thing, I also didn't want to expose the controller directly to the index.js